gui/src/daemon/model.rs

```diff
@@ -129,7 +133,14 @@ impl SpendTx {
             psbt,
             change_indexes,
             spend_amount,
-            fee_amount: inputs_amount - spend_amount - change_amount,
+            // FIXME: this is a quick hack to deal with descendant PSBTs whose ancestor has been replaced.
+            // At least one input coin no longer exists so fee would be negative.
+            // Perhaps such PSBTs should be deleted or hidden?
```

Comment:
I think we should keep them but display a "Can't compute fees" placeholder. xref https://github.com/wizardsardine/liana/issues/820 and https://github.com/wizardsardine/liana/pull/825.